packages/blade/src/components/NavigationBar/_decisions/decisions.md

```diff
@@ -0,0 +1,261 @@
+- [Product Concept Note - Connected Navigation](https://docs.google.com/document/d/1MyCgIS8i3hfhYPiU94oNT0BwqFyNSa8fhVZLT1FlS3s/edit?usp=sharing) (Internal)
+
+## Anatomy & Components
+
+- NavigationBar
+  - NavigationBarBrand *(Logo or Brand Name)*
+  - NavigationBarContent *(Can contain any JSX)*
+    - TabNav *(Horizontal Navigation Bars)*
+      - TabNavItem
+  - NavigationBarActions *(Right Aligned Actions)*
+    - SearchInput
+    - Button
+    - Tooltip
+    - Menu
+      - Avatar
+      - MenuOverlay
+
+## Basic Usage
+
+```jsx
+// Desktop Navigation Bar
+<NavigationBar>
+  <NavigationBarBrand>
+    <img src="razorpay-logo" />
+  </NavigationBarBrand>
+  <NavigationBarContent>
+    <TabNav>
+      <TabNavItem as={RouterLink} isSelected href="/home" icon={HomeIcon} />
+      <TabNavItem as={RouterLink} href="/money">Money</TabNavItem>
+      <TabNavItem as={RouterLink} href="/payroll">Payroll</TabNavItem>
+      <Menu>
+        <TabNavItem trailingIcon={ChevronDown}>Payroll</TabNavItem>
+        <MenuOverlay>
+          { /* Other Menu Items */ }
+        </MenuOverlay>
+      </Menu>
+    </TabNav>
+  </NavigationBarContent>
+  <NavigationBarActions>
+    <SearchInput />
+    <Tooltip content="View Ecosystem Health">
+      <Button icon={ActivityIcon} />
+    <Tooltip>
+    <Tooltip content="View Announcements">
+      <Button icon={AnnouncementIcon} />
+    </Tooltip>
+    <Tooltip content="Ask Ray">
+      <Button icon={RayIcon} />
+    </Tooltip>
+    <Menu>
+      <Avatar />
+      <MenuOverlay>
+        { /* Avatar Menu Items */ }
+      </MenuOverlay>
+    </Menu>
+  </NavigationBarActions>
+</NavigationBar>
+
+// Mobile Navigation Bar
+<NavigationBar>
+  <NavigationBarContent>
+    <Button icon={MenuIcon} variant="secondary" accessibilityLabel="Toggle side navigation" />
+    <Text>{currentPageTitle}</Text>
+  </NavigationBarContent>
+  <NavigationBarActions>
+    <SearchInput />
+    <Button icon={ActivityIcon} accessibilityLabel="View Ecosystem Health" />
+    <Button icon={AnnouncementIcon} accessibilityLabel="View Announcements" />
+    <Button icon={RayIcon} accessibilityLabel="Ask Ray" />
+    <Avatar onClick={openBottomSheet} />
+    <BottomSheet>
+      { /* Avatar BottomSheet Content */ }
+    </BottomSheet>
+  </NavigationBarActions>
+</NavigationBar>
+```
+
+## API
+
+All the top level components like `NavigationBar`, `NavigationBarBrand`, `NavigationBarContent`, `NavigationBarActions` accepts `children` prop which can be any JSX element.
+
+```ts
+type NavigationBarX = {
+  children: React.ReactNode;
+};
+```
+
+### NavigationBar
+
+The top-level container for the navigation bar.
+
+### NavigationBarBrand
+
+The brand logo or name that appears on the left side of the navigation bar.
+
+### NavigationBarContent
+
+The container for the main navigation items.
+
+### NavigationBarActions
+
+The container for the right-aligned actions like search, buttons, and user profile.
+
+### TabNav
```

Comment:
> When would they use TabNav outside of TopNav?

Some usecases similar to [github](https://primer.style/components/tab-nav).

But it may or may not be used. I don't see a real benefit also on preventing them to do so.